### ISSUE TITLE
Incorrect configuration in brms installation guide for Mac M1

### ISSUE THREAD

**Issue by ismailcherri** [learnB4SS/learnB4SS#80]
Hi!

In the Install brms guide, the section about [Configure the C++ toolchain
](https://learnb4ss.github.io/learnB4SS/articles/install-brms.html#configure-the-c-toolchain-2) optimization, the last step contains wrong configuration for Mac M1 computers. The `-arch` value should be `arm64` not `x86_64`

```
dotR <- file.path(Sys.getenv("HOME"), ".R")
if (!file.exists(dotR)) dir.create(dotR)
M <- file.path(dotR, "Makevars")
if (!file.exists(M)) file.create(M)
cat("\nCXX14FLAGS += -O3 -mtune=native -arch arm64 -ftemplate-depth-256",
    file = M, sep = "\n", append = FALSE)
```

This made me spend a lot of time trying to figure out why it is not working on my machine.

**Comment by stefanocoretta** [comment]
Hi! Thanks for spotting this. Those instructions are out of date (they were written when the workshop was delivered a few years ago). I will remove that part since it is not needed anymore.

**Comment by stefanocoretta** [comment]
Fixed in c23d288
